docs/tutorials/self-hosting.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 25
+title: Self-Hosting
+description: Learn how to self-host DBOS Transact applications
+---
+
+You can run DBOS applications anywhere with `npx dbos start` as long as they have a Postgres database to connect to.
+This guide describes tools that make it easy to self-host DBOS applications in production.
```

Comment:
```suggestion
This guide describes tools you can use in your hosting environment to make the most of DBOS transact.
```
These tools don't make it easy to host DBOS apps, but rather allows one to hook with all the cool stuff DBOS transact offer (namely: workflow recovery and built-in observability.)